.github/workflows/publish.yml

```
@@ -1,15 +1,18 @@
 name: publish
-env:
-  DEBUG: napi:*
-  NX_RUN_GROUP: ${{ github.run_id }}-${{ github.run_attempt }}
-  NPM_CONFIG_PROVENANCE: true
+
 on:
+  schedule:
+    - cron: "0 3 * * *" # Once per day, at 3am UTC
```

Can we have this for weekdays only? 

Run Tuesday - Saturday (Picks up changes from Monday - Friday).

```suggestion
    - cron: "0 3 * * 2-6" # Tuesdays - Saturdays, at 3am UTC
```

I don't think we're going to land many changes over the weekends we'll just end up with the same version with different dates.